CMakeLists.txt

```
@@ -404,6 +404,17 @@ if(CRYPTO_BACKEND STREQUAL "WinCNG" OR NOT CRYPTO_BACKEND)
     set(CRYPTO_BACKEND_INCLUDE_DIR "")
     list(APPEND LIBRARIES "crypt32" "bcrypt")
     list(APPEND LIBSSH2_PC_LIBS_PRIVATE "-lcrypt32" "-lbcrypt")
+    
+    option(ENABLE_ECDSA_WINCNG "WinCNG ECDSA support" OFF)
```

```suggestion
    option(ENABLE_ECDSA_WINCNG "WinCNG ECDSA support (requires Windows 10 or later)" OFF)
```
What do you think of adding this info?